notes/vim/copypaste.md

```diff
@@ -1,4 +1,15 @@
-## VIM Copy and Paste tricks
+## Copy and Paste
 
-Tricks here
+y=yank; d=delete; c=cut (which is "delete then insert mode")
 
+These are used for the start of a sequence, followed by what to yank, delete or cut.
+
+Examples:
+
+> yiw : yank inner word
+> dta : delete until the letter 'a'
+
+Double letters (yy, dd, etc) mean: do it for the entire line
+
+p=paste
+P=Paste above
```

Comment:
P=paste before. Using `P` to paste anything less than a line will paste before the cursor on the same line, not above.